### PR TITLE
docs(pattern): import pattern

### PR DESCRIPTION
**Pull request by connor-leech**
This pattern was a contribution. Original PR here:

carbon-design-system/carbon-website-archive#1711

### CODE REVIEW

**Comment by vpicone** [comment]
#### 📝Changed routes:


| [**…ort-pattern/images/0.5**](https://carbon-website-git-fork-connor-leech-pattern-import.carbon-design-system.now.sh/experimental/import-pattern/images/0.5) | [**…mport-pattern/images/1**](https://carbon-website-git-fork-connor-leech-pattern-import.carbon-design-system.now.sh/experimental/import-pattern/images/1) |
|:-:|:-:|
|<a href="https://carbon-website-git-fork-connor-leech-pattern-import.carbon-design-system.now.sh/experimental/import-pattern/images/0.5">Screenshot of /experimental/import-pattern/images/0.5</a><br /><sup><a href="https://microlink-cdn.s3.amazonaws.com/s/fcU_VgzTB3QHiA4wVMsqYDZOkH4_">(view full size)</a> |<a href="https://carbon-website-git-fork-connor-leech-pattern-import.carbon-design-system.now.sh/experimental/import-pattern/images/1">Screenshot of /experimental/import-pattern/images/1</a><br /><sup><a href="https://microlink-cdn.s3.amazonaws.com/s/wANO1_nZpDuflazzRJRq47Svbxk_">(view full size)</a> |



| [**…mport-pattern/images/2**](https://carbon-website-git-fork-connor-leech-pattern-import.carbon-design-system.now.sh/experimental/import-pattern/images/2) | [**…mport-pattern/images/3**](https://carbon-website-git-fork-connor-leech-pattern-import.carbon-design-system.now.sh/experimental/import-pattern/images/3) |
|:-:|:-:|
|<a href="https://carbon-website-git-fork-connor-leech-pattern-import.carbon-design-system.now.sh/experimental/import-pattern/images/2">Screenshot of /experimental/import-pattern/images/2</a><br /><sup><a href="https://microlink-cdn.s3.amazonaws.com/s/iyvul5M11784i95GTUn9aKlKbWg_">(view full size)</a> |<a href="https://carbon-website-git-fork-connor-leech-pattern-import.carbon-design-system.now.sh/experimental/import-pattern/images/3">Screenshot of /experimental/import-pattern/images/3</a><br /><sup><a href="https://microlink-cdn.s3.amazonaws.com/s/so7Ktt6jBQTs5VzUydPzrn7Fz20_">(view full size)</a> |



| [**…mport-pattern/images/4**](https://carbon-website-git-fork-connor-leech-pattern-import.carbon-design-system.now.sh/experimental/import-pattern/images/4) | [**…mport-pattern/images/5**](https://carbon-website-git-fork-connor-leech-pattern-import.carbon-design-system.now.sh/experimental/import-pattern/images/5) |
|:-:|:-:|
|<a href="https://carbon-website-git-fork-connor-leech-pattern-import.carbon-design-system.now.sh/experimental/import-pattern/images/4">Screenshot of /experimental/import-pattern/images/4</a><br /><sup><a href="https://microlink-cdn.s3.amazonaws.com/s/Pi6G3RgW4iOfExPZPXX6I_J0Wvs_">(view full size)</a> |<a href="https://carbon-website-git-fork-connor-leech-pattern-import.carbon-design-system.now.sh/experimental/import-pattern/images/5">Screenshot of /experimental/import-pattern/images/5</a><br /><sup><a href="https://microlink-cdn.s3.amazonaws.com/s/khsVhVr8c_05UtoxJFhNJvyAnFU_">(view full size)</a> |



And 1 other route:
- [**/experimental/import-pattern**](https://carbon-website-git-fork-connor-leech-pattern-import.carbon-design-system.now.sh/experimental/import-pattern)



Commit ecbab8207039a9fc74d0c0f0b95d40af2a6fd16f (https://carbon-website-git-fork-connor-leech-pattern-import.carbon-design-system.now.sh).

File: src/pages/experimental/import-pattern/index.mdx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+#### Status:
+
+[Experimental](experimental/about)
+
+#### Maintainers:
+
+[Vikki Paterson](https://github.com/vikkipaterson)
+
+<AnchorLinks>
```
Comment (janchild) [inline src/pages/experimental/import-pattern/index.mdx]:
```suggestion

```

File: src/pages/experimental/import-pattern/index.mdx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+<AnchorLink>Choose a file</AnchorLink>
```
Comment (janchild) [inline src/pages/experimental/import-pattern/index.mdx]:
```suggestion
```

File: src/pages/experimental/import-pattern/index.mdx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+<AnchorLink>Enter a URL</AnchorLink>
```
Comment (janchild) [inline src/pages/experimental/import-pattern/index.mdx]:
```suggestion

```

File: src/pages/experimental/import-pattern/index.mdx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+</AnchorLinks>
```
Comment (janchild) [inline src/pages/experimental/import-pattern/index.mdx]:
```suggestion

```

File: src/pages/experimental/import-pattern/index.mdx [hunk 5]
```diff
@@ -0,0 +1,76 @@
+---
+title: Import pattern
+---
+
+<PageDescription>
+
+Importing a resource transfers data or objects into a system from an external source.
```
Comment (janchild) [inline src/pages/experimental/import-pattern/index.mdx]:
```suggestion
The import action transfers data or objects from an external source into a system.
```

File: src/pages/experimental/import-pattern/index.mdx [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+
+
+![Example of an import modal in context](/images/0.5.png)
+
+<Caption>Example of an import modal in context</Caption>
+
+## Choose a file
+
+Wherever possible there should always be a drag and drop zone. Use a file drop component, and offer the option to browse locally and select a file. Note that file selection should be restricted to allowed file types.
```
Comment (janchild) [inline src/pages/experimental/import-pattern/index.mdx]:
```suggestion
Wherever possible there should be a drag and drop zone. Use a file drop component and offer the option to browse locally to select a file. Note that file selection should be restricted to allowed file types.
```

File: src/pages/experimental/import-pattern/index.mdx [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+
+<Row>
+<Column colLg={8}>
+
+![Example of an import dialog](/images/2.png)
```
Comment (janchild) [inline src/pages/experimental/import-pattern/index.mdx]:
```suggestion
![Example of an import modal](/images/2.png)
```

File: src/pages/experimental/import-pattern/index.mdx [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+<Caption>Example of an import modal in context</Caption>
+
+## Choose a file
+
+Wherever possible there should be a drag and drop zone. Use a file drop component and offer the option to browse locally to select a file. Note that file selection should be restricted to allowed file types.
+
+<Row>
+<Column colLg={8}>
+
+![Example of an import dialog](/images/2.png)
```
Comment (connor-leech) [inline src/pages/experimental/import-pattern/index.mdx]:
```suggestion
![Example of an import modal](/images/2.png)
```

Comment (janchild) [inline src/pages/experimental/import-pattern/index.mdx]:
```suggestion
![Example of an import modal](/images/2.png)
```

Comment (connor-leech) [inline src/pages/experimental/import-pattern/index.mdx]:
```suggestion
![Example of an import modal](/images/2.png)
```

**Pull request merged**